code/modules/antagonists/traitor/contractor/datums/syndicate_contract.dm

```diff
@@ -461,7 +461,7 @@
 	// Narrate their kidnapping and torturing experience.
 	if(M.stat != DEAD)
 		// Heal them up - gets them out of crit/soft crit.
-		M.reagents.add_reagent("omnizine", 20)
+		M.reagents.add_reagent("omnizine", 10)
```

Comment:
is there a chance this could leave them with an addiction? I guess it makes sense thematically for them to be doubling over in pain

Reply:
It wouldnt be lethal and the syndicate over-prescribing drugs to its prisoners as a last "screw you" seemed comical. Nothing a sleeper wont fix.